auraescript/build.rs

```diff
@@ -1,9 +1,16 @@
+use deno_runtime::snapshot;
 use std::fs::OpenOptions;
 use std::io::Write;
 use std::path::PathBuf;
 
 fn main() {
     generate_aurae_ts();
+
+    // Create a runtime snapshot for Deno to be used on startup
+    snapshot::create_runtime_snapshot(
```

Comment:
Snapshots are now "required" in Deno without passing a couple feature flags to load javascript sources for bootstrapping. This decreases startup performance pretty noticeably, so I went with the snapshot approach. Some info in [this issue](https://github.com/denoland/deno/issues/21928).

```
# With snapshot
» time ./examples/discovery_health.ts 
[Object: null prototype] { healthy: true, version: "0.0.0" }
./examples/discovery_health.ts  0.05s user 0.00s system 75% cpu 0.067 total

# Without snapshot
» time ./examples/discovery_health.ts
[Object: null prototype] { healthy: true, version: "0.0.0" }
./examples/discovery_health.ts  1.82s user 0.02s system 99% cpu 1.847 total
```